examples/google-adk/Getting_started_with_ADK.ipynb

```diff
@@ -0,0 +1,468 @@
+            "'\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## 5. Analyze the Results\n",
+        "\n",
+        "Examine the \"Agent Event Log\" printed above. You should clearly see the sequence reflecting the agent's instructions:\n",
+        "\n",
+        "1.  `Function Call` event targeting `set_workflow_state` with `args={'state_name': 'PROCESSING'}`.\n",
+        "2.  `Function Response` event confirming the first tool execution.\n",
+        "3.  `Function Call` event targeting `set_workflow_state` with `args={'state_name': 'END'}`.\n",
+        "4.  `Function Response` event confirming the second tool execution.\n",
+        "5.  final event containing the agent's text response (the echoed message).\n",
+        "\n",
+        "The next cell verifies this outcome by checking the final state stored in the session."
+      ],
+      "metadata": {
+        "id": "QNDxGtxI0cCR"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "final_session = session_service.get_session(\n",
+        "    session_id=session.id,\n",
+        "    user_id=USER_ID,\n",
+        "    app_name=APP_NAME\n",
+        ")\n",
+        "\n",
+        "if final_session:\n",
+        "    final_state = final_session.state\n",
+        "    workflow_status = final_state.get('workflow_status')\n",
+        "    print(f\"Final workflow status: {workflow_status}\")\n",
+        "\n",
+        "if final_agent_response_text:\n",
+        "    print(f\"Agent response: {final_agent_response_text}\")"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "LJaeajJu0ce0",
+        "outputId": "078bbb62-6328-41b8-b8e9-dc265600f8c0"
+      },
+      "execution_count": 9,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Final workflow status: END\n",
+            "Agent response: Hello ADK!\n",
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Next Steps & Further Learning\n",
+        "\n",
+        "This notebook demonstrated the basic structure of an ADK application, including:\n",
+        "\n",
+        "*   Defining an `Agent` powered by Gemini.\n",
+        "*   Creating a simple `FunctionTool` to modify state.\n",
+        "*   Using `SessionService` and `ToolContext` for state management.\n",
+        "*   Orchestrating the flow with the `Runner`.\n",
+        "\n",
+        "To dive deeper into the capabilities of the Google Agent Development Kit:\n",
+        "\n",
+        "1.  **Explore the Official Documentation:** For detailed explanations of all components (Agents, Tools, Sessions, Callbacks, Multi-Agent systems, etc.), visit the [**Google ADK Documentation site**](https://google.github.io/adk-docs/).\n",
```

Comment:
I'd add a link to their get started notebook in particular: https://colab.sandbox.google.com/github/google/adk-docs/blob/main/examples/python/notebooks/adk_tutorial.ipynb